### PR TITLE
Add myself to the list of students

### LINKED ISSUE #3: Add myself (@ShadySeko) to the list of students
I would like to add myself to the list of students in the GitHub
organization README.

### PR DESCRIPTION
**Pull request by ShadySeko**
Add myself to the list of students

Fixes #3 :)

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -24,4 +24,6 @@ Please add your name in alphabetical order (by last name) in this format:
 First name Last name [@GitHub username](https://github.com/USERNAME)
 -->
 
+
 - _No student yet!_
```
Comment (hadrylouis) [inline profile/README.md]: You can delete this line as there is a student now (you!) :)

**Review by hadrylouis — DISMISSED** [review]

**Comment by hadrylouis** [comment]
Don't forget to associate the PR to its issue, you can do it from the right panel. :)

**Comment by ShadySeko** [comment]
> Don't forget to associate the PR to its issue, you can do it from the right panel. :)

I have no access to modification of the right panel somehow :(

**Comment by hadrylouis** [comment]
Yes sorry for this. Could you please add a comment with the link to your issue with #xxx
You can follow the [create a pull request](https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/03-git-github-and-markdown/COURSE_MATERIAL.md#create-a-pull-request) section of the course material

**Comment by ShadySeko** [comment]
> Yes sorry for this. Could you please add a comment with the link to your issue with #xxx You can follow the [create a pull request](https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/03-git-github-and-markdown/COURSE_MATERIAL.md#create-a-pull-request) section of the course material


I edited the original request to link it to my issue, I guess my pull request is ready now.

**Review by ludelafo — APPROVED** [review]

**Pull request merged**